doc/contributing/commit-queue.md

```diff
@@ -54,6 +54,8 @@ of the commit queue:
 2. A CI must've ran and succeeded since the last change on the PR
 3. A collaborator must have approved the PR since the last change
 4. Only Jenkins CI and GitHub Actions are checked (V8 CI and CITGM are ignored)
+5. The PR must target the `main` branch (PRs opened against other branches, such
+   as backport PRs, are ignored)
```

Comment:
Doesn't have to be resolved in this PR, but the use of an ordered list doesn't really make sense here; the style guide we follow says we should use unordered list, and add periods.
```suggestion
- A CI must've ran and succeeded since the last change on the PR.
- A collaborator must have approved the PR since the last change.
- Only Jenkins CI and GitHub Actions are checked (V8 CI and CITGM are ignored).
- The PR must target the `main` branch (PRs opened against other branches, such
   as backport PRs, are ignored).
```